tests/test_ebuild.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from superflore.generators.ebuild.ebuild import Ebuild
+from superflore.generators.ebuild.ebuild import ebuild_keyword
+from superflore.exceptions import UnresolvedDependency
+import unittest
+
+
+class TestEbuildOutput(unittest.TestCase):
+    def test_simple(self):
+        """Test Ebuild Format"""
+        ebuild = Ebuild()
+        ebuild.homepage = 'https://www.website.com'
+        ebuild.description = 'an ebuild'
+        ebuild.add_run_depend('p2os_driver')
+        ebuild.src_uri = 'https://www.website.com/download/stuff.tar.gz'
+        ebuild.distro = 'lunar'
```

Comment:
The above 6 lines look like boilerplate that could easily be filled into a helper function to avoid repeating them below.